### DISCUSSION TITLE
Asset Tag (Prefix) - Proposal to Expand Tag Prefix Options

### DISCUSSION THREAD

**Discussion by miguelalmeidamiranda** [snipe/snipe-it#14727] [Ideas]
Dear Snipe IT Development Team,

I hope this message finds you well. I am a dedicated user of Snipe IT, utilizing it extensively to manage assets within my company. While the platform has been incredibly beneficial for our asset management needs, I have encountered a limitation regarding tag prefix options.

Currently, Snipe IT allows users to utilize only one tag prefix for categorizing assets. However, in our diverse operational environment, it would greatly enhance efficiency if we could have the flexibility to add 2 or 3 additional tag prefixes.

These additional prefixes would enable us to better organize and differentiate between various types of assets, enhancing the overall management experience within Snipe IT.

I understand that implementing such a feature may require development resources and consideration from your team. Nonetheless, I believe it would greatly benefit not only my organization but also other Snipe IT users who encounter similar challenges in asset categorization.

Therefore, I kindly propose the expansion of tag prefix options within Snipe IT and humbly request that this suggestion be considered for future updates or releases. I am more than willing to provide further insight or assistance to support the implementation of this feature.

Thank you for your attention to this matter, and I look forward to the possibility of seeing this enhancement in Snipe IT's functionality.

Best regards,
Carlos Miranda
Fabamaq

**Comment by tiagofsferreira** [comment]
++ 1 Waiting this feacture
